### ISSUE TITLE
.Sites.First was deprecated

### ISSUE THREAD

**Issue by orsinium** [alex-shpak/hugo-book#645]
When building a hugo-book-powered website with hugo 0.133.0:

```
WARN  deprecated: .Sites.First was deprecated in Hugo v0.127.0 and will be removed in a future release. Use .Sites.Default instead.
```

I believe it concerns the `layouts/partials/docs/brand.html` file.

**Comment by ShaunaGordon** [comment]
There's a PR already for it, it just needs merged in by the maintainer - #622

**Comment by alex-shpak** [comment]
Fixed via #653
